edx/analytics/tasks/tests/test_course_video.py

```diff
@@ -0,0 +1,549 @@
+"""
+Tests for tasks that calculate course videos.
+
+"""
+import json
+import StringIO
+import math
+from operator import itemgetter
+from datetime import datetime
+
+from mock import Mock, patch
+
+from edx.analytics.tasks.course_video import (
+    CourseVideoEventMixin,
+    CourseVideoSummaryTask,
+    CourseVideoSeekTimesTask,
+    UserVideoSummaryTask,
+    CourseVideoWorkflow,
+    round_to_nearest,
+    round_datetime_to_nearest_day,
+    EVENT_ACTION_MAPPING,
+    SEEK_INTERVAL_IN_SEC,
+    DATE_FMT,
+)
+
+from edx.analytics.tasks.tests import unittest
+from edx.analytics.tasks.tests.config import with_luigi_config, OPTION_REMOVED
+from edx.analytics.tasks.tests.opaque_key_mixins import InitializeOpaqueKeysMixin
+
+
+VIDEO_MAX_LEN_IN_SEC = 300
```

Comment:
For some tests, it's useful to know the max length of the video, since we can create play/seek/stop log events based on the total length of the video rather than raw numbers.